pi.py

```diff
@@ -12,5 +12,5 @@
     # import newer version
     from mpmath import mp

```

Comment:
Why not just import only the new version?

Reply:
There were some issues with the new version so check on StackO there someone suggested using the old. Hence the error handling